pom.xml

```diff
@@ -342,13 +342,6 @@
                         <skipRuntimeScope>true</skipRuntimeScope>
                     </configuration>
                 </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-assembly-plugin</artifactId>
-                    <configuration>
-                        <skipAssembly>true</skipAssembly>
-                    </configuration>
-                </plugin>
```

Comment:
It's needed, in other way the final assembly packages are missing when releasing a new version of Struts 7 - no idea why it works in different way than in Struts 6

Reply:
Ah, let me revert